crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/prosecution_concluded/service/ProsecutionConcludedServiceTest.java

```diff
@@ -158,16 +171,16 @@ void givenMessageIsReceived_whenHearingDataNotInMaat_thenTriggerHearingProcessin
         verify(prosecutionConcludedValidator).validateRequestObject(any());
         verify(courtDataAPIService, atLeast(1)).retrieveHearingForCaseConclusion(any());
         verify(courtDataAPIService, never()).isMaatRecordLocked(anyInt());
-        verify(prosecutionConcludedImpl, never()).execute(any());
+        verify(prosecutionConcludedImpl, never()).execute(any(), any());
         verify(calculateOutcomeHelper, never()).calculate(any());
-        verify(caseConclusionDTOBuilder, never()).build(any(), any(), any());
+        verify(caseConclusionDTOBuilder, never()).build(any(), any(), any(), any());
         verify(offenceHelper, never()).getTrialOffences(any(), anyInt());
     }
 
     private ProsecutionConcluded getProsecutionConcluded() {
         return ProsecutionConcluded.builder()
                 .isConcluded(true)
-                .maatId(1221)
+                .maatId(1212111)
```

Comment:
I'd prefer if we could use a constant here, usually we create one in the `TestModelDataBuilder` and share it.

Reply:
Done